### DISCUSSION TITLE
Performance mode from Procreate Dreams

### DISCUSSION THREAD

**Discussion by drixuniverse** [keiwando/toonsquid#594] [Ideas]
Since dreams dropped it hasn't leveled up to Toonsquid at all in terms of features, But I have messed around with it and I think performance mode is the most brilliant part of it. There's just a level of fluidity to the movements that you can't match while keyframing manually unless you spend absurd amounts of time adjusting. Toonsquid should have its own rendition of the feature within the keyframe editor.

**Comment by Jebs88** [comment]
And also playback speed for the performance mode.

**Comment by kavlor** [comment]
This would be really great with the inverse kinematics.

**Comment by torazine** [comment]
Performance mode is a feature I would really like to see added.

**Comment by Cr8iveMike** [comment]
I know that a lot will disagree with me on this fact that PDreams is equally as good as ToonSquid in many areas. It's just that many are unable to use its limited feature to a great advantage especially when you don't factor this fact that Procreate & PDreams are meant to be used as one as intended. But since I'm in ToonSquid page I will give my point to TS over PD with this particular feature called Vector. Performance Mode is a joy to have on PD and would love to see that feature added to TS.

**Reply by Rohitgit2412** [reply]
PD is good when it comes to motion graphics no doubt, but in case of traditional frame by frame animation it is very limited and no where near toonsquid because the frame by frame animation requires a totally different approach where keyframed are not much of a help. But PDs interface and learning curve is very simple that's major thing about it. Its only the interface in toondquid that I personally want to see some improvement otherwise it's the best ipad animation app.

**Reply by Cr8iveMike** [reply]
> PD is good when it comes to motion graphics no doubt, but in case of traditional frame by frame animation it is very limited and no where near toonsquid because the frame by frame animation requires a totally different approach where keyframed are not much of a help. But PDs interface and learning curve is very simple that's major thing about it. Its only the interface in toondquid that I personally want to see some improvement otherwise it's the best ipad animation app.

I hear you my friend and I won't argue with you on that as I believe it is all about preference. But I will stand with the fact that they both have their Pros & Cons and I use them equally too. To me PD has nice interface and gestures to go along with....many do not like it but I do. Performance mode, sync well with procreate are points also. And TS has vector features and symbols but the interface is not my go choice. I use it for motion graphics more though because of vector trim path. But in terms of intuitive, I do more with PD leveraging on the Performance Mode.

**Comment by Rohitgit2412** [comment]
Performance mode + recording the drawing strokes. The recording while drawing feature is in callipeg. You have to set the frame numbers within which the strokes will be recorded and the app will automatically record the strokes in succeeding frames based on the number you set. This feature would be helpful when drawing trails/ calligraphy/ motion graphics. One more feature i would like is where you can multiply animation layer several time and set its movement path randomness etc. this feature is in fresco. Example- you can draw one honeybee and rest just add the effect of multiplying it and setting the randomness of movement. This feature would be helpful in scenes like snowfall, rain, falling leaves bugs insects flying where you don't have to go through the pain of drawing lot of same random moving elements.
